other/event_tooling/generate_event_c.cpp

```diff
@@ -686,7 +686,7 @@ int main(int argc, char** argv) {
             {
                 EventTypeTrivial{"uint32_t", "group_number"},
                 EventTypeTrivial{"uint32_t", "peer_id"},
-                EventTypeByteRange{"name", "name_length", "length"}, // the latter two are ideally the same
+                EventTypeByteRange{"name", "name_length", "name_length"}, // they are :)
```

Comment:
Yeah, wtf are these "they are :)" comments, get that out of here!

Reply:
D: